### DISCUSSION TITLE
ignorePattern does not seem to be working

### DISCUSSION THREAD

**Discussion by iknowmagic** [jonaskello/eslint-plugin-functional#465] [Q&A] [answered]
Thank you so much for this great ESLint plugin :)

It seems that ignorePattern is not working.
Here is my eslint rule:
`
'functional/functional-parameters': [
      'error',
      { ignorePattern: ['computed'] }
    ],
`
I have a piece of code that looks like this:
`
const alertClasses = computed(() => {
  ...
})
`
to match agaist the aforemetioned eslint rule. 

Unfortunatelly eslint is not taking into account the ignorePattern from above.

<img width="2022" alt="image" src="https://user-images.githubusercontent.com/8712362/188751317-33866958-3acb-4258-bf66-9450fae84219.png">

**Comment by RebeccaStevens** [comment, marked as answer]
The arrow function declaration is what is being reported as violating the rule; not the call of the `computed` function. The `ignorePattern` option work by matching against the name of the function, but as the arrow function has no name, it cannot be ignored via this option.

I'd recommend either adding an `eslint-disable-next` comment before the allow function or adjusting your config to not enforce a parameter count. I just created #468 which if adding will give another option to approach this.
